content/wiki/backports.md

```diff
@@ -0,0 +1,75 @@
++++
+title = "Backporting in Linebender projects"
++++
+
+## When to backport
+
+Generally we publish the latest state of the `main` branch.
+However, if `main` has breaking changes or we want an older release series to receive some non-breaking changes, then we can't just publish `main`.
+That's when we need to create a backporting branch to achieve that goal.
+
+The example commands below are based on Color and its `main` already being at `v0.3.1` when we also wanted to release a `v0.2.4`, with `v0.2.3` being the latest in that series.
+Make sure you modify the commands based on your actual versions.
+
+## Create the backport branch
+
+The backport branch needs to be based on the most recent release tag in the series.
+It must be named so it matches `v*.x`, e.g. `v0.2.x` or `v1.x.x`.
```

Comment:
We could probably just do `v1.x` once we've hit semver major?

Reply:
As far as the matching goes, yes, it supports it by design. Visually in a list of branches I would prefer to see `v1.x.x` but it's such a tiny detail that I certainly don't hold any strong opinion on that.